rts/motoko-rts/src/gc/mark_compact/bitmap.rs

```diff
@@ -4,37 +4,45 @@ use crate::types::{size_of, Blob, Bytes, Obj};
 
 /// Current bitmap
 static mut BITMAP_PTR: *mut u8 = core::ptr::null_mut();
+static mut BITMAP_COMPENSATION: usize = 0;
+static mut BITMAP_SIZE: u32 = 0;
 
-pub unsafe fn alloc_bitmap<M: Memory>(mem: &mut M, heap_size: Bytes<u32>) {
+pub unsafe fn alloc_bitmap<M: Memory>(mem: &mut M, heap_size: Bytes<u32>, heap_prefix_words: u32) {
+    assert_eq!(heap_prefix_words % 8, 0);
```

Comment:
this can be `debug_` since we `assert!` the harder fact in rts/motoko-rts/src/gc/mark_compact.rs:75 already